samples/meta-codegen/lib/pom.xml

```diff
@@ -116,7 +116,7 @@
     </dependencies>
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-        <openapi-generator-version>3.0.0-SNAPSHOT</openapi-generator-version>
+        <openapi-generator-version>3.2.2-SNAPSHOT</openapi-generator-version>
```

Comment:
I'll take note of this during the release process.

Reply:
`lib/` is generated (output of the `meta` command of the CLI). It is updated with ensure-up-to-date.